src/generators/java/mekanism/generators/common/MekanismGenerators.java

```diff
@@ -112,6 +113,9 @@ private void commonSetup(FMLCommonSetupEvent event) {
             BuildCommand.register("turbine", GeneratorsLang.TURBINE, new TurbineBuilder());
             BuildCommand.register("fission", GeneratorsLang.FISSION_REACTOR, new FissionReactorBuilder());
             BuildCommand.register("fusion", GeneratorsLang.FUSION_REACTOR, new FusionReactorBuilder());
+            if (Mekanism.hooks.CCLoaded) {
+                ComputerMethodRegistry_mekanismgenerators.init();
```

Comment:
The Mekanism Generators one can't go into MekanismHooks as it's not in that source set.
So I'm not quite sure what you're after

Reply:
if that's the goal we could make it use a service loader I guess, and throw a test or runtime exception if it doesnt find any

Reply:
also, what is "enqueueWork block"?

Reply:
`event.enqueueWork` basically queues the body of it to be ran on the main thread. And using service loaders for it sounds like it may make sense, presuming the annotation processor can generate the service loader meta-inf file as well (which I assume it can)

Reply:
I totally didn't notice it was inside the `enqueueWork` block lol

Reply:
I had just assumed because I couldn't remember if strict order definitions for mods would ensure the parallel events ran sequentially rather than at the same time that maybe it was to ensure the mod loaded checks passed (though I do think the service loader solution is a lot cleaner and also is nice that it avoids this)